### PR TITLE
Add support Rails 7.0 in CI.

### PR DESCRIPTION
**Pull request by osyo-manga**

### CODE REVIEW

**Review by mkmn — APPROVED** [review]
👍

File: gemfiles/rails_5.2.gemfile [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
-
```
Comment (osyo-manga) [inline gemfiles/rails_5.2.gemfile:2]: It's gone because Appraisal generates it automatically.

File: .circleci/config.yml [hunk 2]
```diff
@@ -94,8 +106,9 @@ workflows:
       - ruby_2_7_rails_5_2
       - ruby_2_7_rails_6_0
       - ruby_2_7_rails_6_1
+      - ruby_2_7_rails_7_0
       - ruby_2_7_rails_main
-      - ruby_3_0_rails_6_1
```
Comment (wakasa51) [inline .circleci/config.yml:98]: If you don't need `ruby_3_0_rails_6_1`  job definition, it may be better to delete (line 80)

Comment (osyo-manga) [reply .circleci/config.yml:98]: Thanks! Fix it https://github.com/kufu/activerecord-bitemporal/pull/85/commits/9fe1d1561d77aaa86387e58f3839ccb85df0ec75

**Review by wakasa51 — APPROVED** [review]
👍

**Comment by osyo-manga** [comment]
@mkmn @wakasa51 Thanks!

**Pull request merged**